src/AppBundle/Form/ProductType.php

```diff
@@ -95,6 +95,21 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         // While price & tax category are defined in ProductVariant,
         // we display the fields at the Product level
         // For now, all variants share the same values
+        $builder
```

Comment:
The comments on lines 95-97 aren't related to this code block, so this is confusing. 

Also, I'm thinking that those fields should be added only when the restaurant has `depositRefundEnable = true` 🤔 
So, this should be moved inside the `FormEvents::POST_SET_DATA` listener (below)